### ISSUE TITLE
failed to take snapshot

### ISSUE THREAD

**Issue by sethvincent** [developmentseed/observe#72]
When viewing an offline map there's a gray screen where the map snapshot should be and this error is logged:

```
failed to take snapshot: Error: Could not create snapshot
    at createErrorFromErrorData (NativeModules.js:146)
    at NativeModules.js:95
    at MessageQueue.__invokeCallback (MessageQueue.js:414)
    at MessageQueue.js:127
    at MessageQueue.__guard (MessageQueue.js:314)
    at MessageQueue.invokeCallbackAndReturnFlushedQueue (MessageQueue.js:126)
    at debuggerWorker.js:72
```

**Comment by sethvincent** [comment]
So far I've only seen this happen on emulator. If it never happens on device it's low priority, but when demoing the app via emulator it makes things look messed up.

**Comment by geohacker** [comment]
That's coming from https://github.com/developmentseed/observe/blob/d56ba9c33d34b40033b4eebeda0cb67cd100024a/app/screens/OfflineMaps/ViewOfflineAreaDetail.js#L87 and I was only able to reproduce this with iOS. Looks like the asset URL for the style json was incorrect for OSM. 

Submitted a fix in #76
